packages/plugin-ext/src/main/browser/tasks-main.ts

```diff
@@ -234,6 +234,7 @@ export class TasksMainImpl implements TasksMain, Disposable {
         return {
             ...common,
             ...partialDto,
+            taskType: taskType,
```

Comment:
[Minor]: Could be just `taskType`.

Reply:
or https://github.com/eclipse-theia/theia/pull/9377#discussion_r617070847

Reply:
so, it looks like we don't need the change here